index.html

```diff
@@ -289,6 +289,26 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact_form_container">
+        <section class="contact_form_section">
+          <h2 class="contact_form_header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</h2>
+          <div class="form_container">
+            <form class="contact form" action="https://formspree.io/f/xeqvdkea" method="POST">
+              <div class="name">
+                <input type="text" name="Name" id="first_name" placeholder="First name" maxlength="30">
+                <input type="text" name="Surname" id="last_name" placeholder="Last name" maxlength="30">
+              </div>
+              <div class="email">
+                <input type="email" name="Email" id="email_address" placeholder="Email address">
+              </div>
+              <div class="info">
+                <textarea name="Info" id="information" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
```

Comment:
- [ ] **[REQUIRED CHANGES]** As mentioned in the last review, it is important that all of the `inputs` and the `textarea` have the `required` HTML property, in order to make sure that the user won't forget any important information!